hashtheplanet/config/extensions_list.py

```diff
@@ -0,0 +1,4 @@
+"""
+This file contains a list of extensions to exclude, allowing HashThePlanet to take only interesting files to hash
+"""
```

Comment:
The comment is no longer accurate: this is a regular expression, not a list of extensions to exclude. I could have named the variable `EXCLUDED_FILE_PATTERN` or something.

Reply:
Fixed